streaming/src/main/scala/org/apache/spark/streaming/receiver/Receiver.scala

```diff
@@ -83,7 +84,12 @@ import org.apache.spark.storage.StorageLevel
  * }}}
  */
 @DeveloperApi
-abstract class Receiver[T](val storageLevel: StorageLevel) extends Serializable {
+abstract class Receiver[T](val storageLevel: StorageLevel)(implicit t: ClassTag[T])
```

Comment:
I'm wondering will this change the signature of `Receiver` and break the existing code by user?